privacyidea/lib/smtpserver.py

```diff
@@ -113,6 +115,10 @@ def test_email(config, recipient, subject, body, sender=None,
             password = decryptPassword(config['password'])
             if password == FAILED_TO_DECRYPT_PASSWORD:
                 password = config['password']
+            # Under Python 2, we pass passwords as bytestrings to get CRAM-MD5 to work.
+            # Under Python 3, we pass passwords as unicode.
+            if six.PY2:
+                password = to_utf8(password)
```

Comment:
Maybe use `to_bytes()`, it basically does the same and the name is more intuitive (i think).
I think we should remove `to_utf8()` in the long run.

Reply:
Yes, will do!

Reply:
I changed ``to_utf8`` to ``to_bytes`` in c298946.

Reply:
thanks, @plettich for the explanation. So this can be merged.